.github/workflows/coding_standard.yml

```diff
@@ -1,89 +0,0 @@
-name: Coding standard
```

Comment:
To be honest we're splitting the builds for build.yaml and coding_standard.yaml currently, but I can see the current coding_standard.yml file would need to be redefined, so I can agree with removing it.